src/Sulu/Bundle/AdminBundle/Resources/js/components/Toolbar/types.js

```diff
@@ -8,6 +10,9 @@ export type Item =
     Element<typeof ButtonComponent>
     | Element<typeof DropdownComponent>
     | Element<typeof SelectComponent>;
+export type Group = Element<typeof ItemsComponent> | Element<typeof IconsComponent>;
+
+export type Skins = 'light' | 'dark';
```

Comment:
Until now we have also used Singular for type definitions like this, so it would be `Skin` to stay consistent.